subprojects/plugins/src/main/java/org/gradle/api/plugins/jvm/internal/DefaultJvmTestSuite.java

```diff
@@ -50,7 +50,7 @@ public enum Frameworks {
         JUNIT4("junit:junit", "4.13"),
         JUNIT_JUPITER("org.junit.jupiter:junit-jupiter", "5.7.2"),
         SPOCK("org.spockframework:spock-core", "2.1-groovy-3.0"),
-        KOTLIN_TEST("org.jetbrains.kotlin:kotlin-test-junit", "1.6.21"),
+        KOTLIN_TEST("org.jetbrains.kotlin:kotlin-test-junit5", "1.6.21"),
         TESTNG("org.testng:testng", "7.4.0"),
```

Comment:
As a separate PR, could you make a pass at updating the versions here?

Reply:
Will do